source/common/grpc/google_grpc_creds_impl.cc

```diff
@@ -11,15 +11,23 @@ namespace Grpc {
 
 std::shared_ptr<grpc::ChannelCredentials> CredsUtility::sslChannelCredentials(
```

Comment:
This method need a better name, it is not only about SSL with this change.

Reply:
Renamed to getChannelCredentials(). Thanks.